frontend/src/pages/secret-manager/SecretDashboardPage/components/SnapshotView/SecretItem.tsx

```diff
@@ -76,7 +76,7 @@ export const SecretItem = ({ mode, preSecret, postSecret }: Props) => {
           <FontAwesomeIcon icon={faKey} />
         </div>
         <div className="flex flex-grow items-center space-x-4 px-4 py-3">
-          {mode === "modified" ? (
+          {mode === "modified" && !preSecret?.isRotatedSecret ? (
```

Comment:
not sure why but even when I only made changes to HELOZ, the rotated secrets are tagged as "deleted"
![Uploading image.png…]()

maybe it should just be "no change" always if its a rotated secret?